test/functional/helpers.lua

```diff
@@ -28,7 +28,7 @@ local nvim_prog = (
 -- Default settings for the test session.
 local nvim_set  = 'set shortmess+=IS background=light noswapfile noautoindent'
                   ..' laststatus=1 undodir=. directory=. viewdir=. backupdir=.'
-                  ..' belloff= noshowcmd noruler nomore'
+                  ..' belloff= wildoptions= noshowcmd noruler nomore'
```

Comment:
Why isn't it using the default with tests?

Reply:
because it will break existing tests. We have tests for both behaviours anyway.

Reply:
`wildoptions=pum` is rarely useful for most screen tests because it would cover the buffer contents. Maybe I'll change this to `wildoptions-=pum`. I wouldn't be surprised if `tagfile` has no test coverage at all.